charts/posit-chronicle/templates/configmap.yaml

```diff
@@ -27,7 +27,7 @@ data:
 
     [Profiling]
     Enabled = {{ .Values.config.Profiling.Enabled }}
-    Listen = 3030
+    Listen = :3030
```

Comment:
This would cause the a `http server error: listen tcp: address 3030: missing port in address` error log at bootup if you were using the default chart values from `0.1.0`

```
{"level":"info","msg":"starting debug profiling server, listening at 3030","time":"2023-12-05T19:33:53.781Z"}
{"level":"error","msg":"http server error: listen tcp: address 3030: missing port in address","time":"2023-12-05T19:33:53.781Z"}
{"level":"info","msg":"storing metrics in parquet files in S3 at posit-chronicle-dev/v1/metrics","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"storing metrics in parquet files at chronicle-data/v1/metrics","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"storing logs in parquet files in S3 at posit-chronicle-dev/v1/logs","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"storing users in parquet files in S3 at posit-chronicle-dev/v1/users","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"storing logs in parquet files at chronicle-data/v1/logs","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"storing users in parquet files at chronicle-data/v1/users","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"enabling Prometheus /metrics endpoint","time":"2023-12-05T19:33:53.782Z"}
{"level":"info","msg":"starting http server, listening at :5252","time":"2023-12-05T19:33:53.782Z"}

```

This was easy to miss as the error won't crash the server, and we were overwriting this value with `:3030` in the ci values we used for templating and QA.